host/xtest/regression_4000.c

```diff
@@ -1009,6 +1009,172 @@ static const uint8_t hash_data_shake256_out1[] = {
+	0x19, 0x85, 0x0a, 0x50, 0x6d, 0xfa, 0x9b, 0x0d, 0x25, 0x47, 0x95, 0xbe, 0x78, 0x57, 0x86, 0xc3,
+	0x19, 0xa2, 0x56, 0x7c, 0xba, 0xa5, 0xe3, 0x5b, 0xcf, 0x8f, 0xe8, 0x3d, 0x94, 0x3e, 0x23, 0xfa,
+	0x51, 0x69, 0xb7, 0x3a, 0xdc, 0x1f, 0xcf, 0x8b, 0x60, 0x70, 0x84, 0xb1, 0x5e, 0x6a, 0x01, 0x3d,
+	0xf1, 0x47, 0xe4, 0x62, 0x56, 0xe4, 0xe8, 0x03, 0xab, 0x75, 0xc1, 0x10, 0xf7, 0x78, 0x48, 0x13,
+	0x6b, 0xe7, 0xd8, 0x06, 0xe8, 0xb2, 0xf8, 0x68, 0xc1, 0x6c, 0x3a, 0x90, 0xc1, 0x44, 0x63, 0x40,
+	0x70, 0x38, 0xcb, 0x7d, 0x92, 0x85, 0x07, 0x9e, 0xf1, 0x62, 0xc6, 0xa4, 0x5c, 0xed, 0xf9, 0xc9,
+	0xf0, 0x66, 0x37, 0x5c, 0x96, 0x9b, 0x5f, 0xcb, 0xcd, 0xa3, 0x7f, 0x02, 0xaa, 0xcf, 0xf4, 0xf3,
+	0x1c, 0xde, 0xd3, 0x76, 0x75, 0x70, 0x88, 0x54, 0x26, 0xbe, 0xbd, 0x9e, 0xca, 0x87, 0x7e, 0x44,
+	0x67, 0x4e, 0x9a, 0xe2, 0xf0, 0xc2, 0x4c, 0xdd, 0x0e, 0x7e, 0x1a, 0xaf, 0x1f, 0xf2, 0xfe, 0x7f,
+	0x80, 0xa1, 0xc4, 0xf5, 0x07, 0x8e, 0xb3, 0x4c, 0xd4, 0xf0, 0x6f, 0xa9, 0x4a, 0x2d, 0x1e, 0xab,
+	0x58, 0x06, 0xca, 0x43, 0xfd, 0x0f, 0x06, 0xc6, 0x0b, 0x63, 0xd5, 0x40, 0x2b, 0x95, 0xc7, 0x0c,
+	0x21, 0xea, 0x65, 0xa1, 0x51, 0xc5, 0xcf, 0xaf, 0x82, 0x62, 0xa4, 0x6b, 0xe3, 0xc7, 0x22, 0x26,
+	0x4b
+};
+
+static const uint8_t hash_data_sha3_384_out2[] = {
+	0x30, 0x54, 0xd2, 0x49, 0xf9, 0x16, 0xa6, 0x03, 0x9b, 0x2a, 0x9c, 0x3e, 0xbe, 0xc1, 0x41, 0x87,
+	0x91, 0xa0, 0x60, 0x8a, 0x17, 0x0e, 0x6d, 0x36, 0x48, 0x60, 0x35, 0xe5, 0xf9, 0x26, 0x35, 0xea,
+	0xba, 0x98, 0x07, 0x2a, 0x85, 0x37, 0x3c, 0xb5, 0x4e, 0x2a, 0xe3, 0xf9, 0x82, 0xce, 0x13, 0x2b
+};
+
+static const uint8_t hash_data_sha3_512_in2[] = {
+	0x66, 0x4e, 0xf2, 0xe3, 0xa7, 0x05, 0x9d, 0xaf, 0x1c, 0x58, 0xca, 0xf5, 0x20, 0x08, 0xc5, 0x22,
+	0x7e, 0x85, 0xcd, 0xcb, 0x83, 0xb4, 0xc5, 0x94, 0x57, 0xf0, 0x2c, 0x50, 0x8d, 0x4f, 0x4f, 0x69,
+	0xf8, 0x26, 0xbd, 0x82, 0xc0, 0xcf, 0xfc, 0x5c, 0xb6, 0xa9, 0x7a, 0xf6, 0xe5, 0x61, 0xc6, 0xf9,
+	0x69, 0x70, 0x00, 0x52, 0x85, 0xe5, 0x8f, 0x21, 0xef, 0x65, 0x11, 0xd2, 0x6e, 0x70, 0x98, 0x89,
+	0xa7, 0xe5, 0x13, 0xc4, 0x34, 0xc9, 0x0a, 0x3c, 0xf7, 0x44, 0x8f, 0x0c, 0xae, 0xec, 0x71, 0x14,
+	0xc7, 0x47, 0xb2, 0xa0, 0x75, 0x8a, 0x3b, 0x45, 0x03, 0xa7, 0xcf, 0x0c, 0x69, 0x87, 0x3e, 0xd3,
+	0x1d, 0x94, 0xdb, 0xef, 0x2b, 0x7b, 0x2f, 0x16, 0x88, 0x30, 0xef, 0x7d, 0xa3, 0x32, 0x2c, 0x3d,
+	0x3e, 0x10, 0xca, 0xfb, 0x7c, 0x2c, 0x33, 0xc8, 0x3b, 0xbf, 0x4c, 0x46, 0xa3, 0x1d, 0xa9, 0x0c,
+	0xff, 0x3b, 0xfd, 0x4c, 0xcc, 0x6e, 0xd4, 0xb3, 0x10, 0x75, 0x84, 0x91, 0xee, 0xba, 0x60, 0x3a,
+	0x76
+};
+
+static const uint8_t hash_data_sha3_512_out2[] = {
+	0xe5, 0x82, 0x5f, 0xf1, 0xa3, 0xc0, 0x70, 0xd5,
+	0xa5, 0x2f, 0xbb, 0xe7, 0x11, 0x85, 0x4a, 0x44,
+	0x05, 0x54, 0x29, 0x5f, 0xfb, 0x7a, 0x79, 0x69,
+	0xa1, 0x79, 0x08, 0xd1, 0x01, 0x63, 0xbf, 0xbe,
+	0x8f, 0x1d, 0x52, 0xa6, 0x76, 0xe8, 0xa0, 0x13,
+	0x7b, 0x56, 0xa1, 0x1c, 0xdf, 0x0f, 0xfb, 0xb4,
+	0x56, 0xbc, 0x89, 0x9f, 0xc7, 0x27, 0xd1, 0x4b,
+	0xd8, 0x88, 0x22, 0x32, 0x54, 0x9d, 0x91, 0x4e,
+};
+
+static const uint8_t hash_data_shake128_in2[] = {
```

Comment:
Where did you find this test vector?

Reply:
Nist website:
https://csrc.nist.gov/projects/cryptographic-algorithm-validation-program/secure-hashing

Reply:
Done.  Comments updated with where the test vectors came from.  Thanks.